### ISSUE TITLE
[TASK] Allow events with no enddate

### ISSUE THREAD

**Issue by derhansen** [derhansen/sf_event_mgt#880]
In #849 the start and enddate changed to be required fields. This change should only have been applied to the `startdate` field, since the `enddate` can also be empty.

**Comment by terminal8-nz** [comment]
This seems to lead to problems with the display modes. If I have events without `enddate`, they will always show if I select `past` (because afaik the `startdate` is not checked in this case). Can you confirm that this is not the intended behavior?

**Comment by derhansen** [comment]
Yes, that is a bug and I already fixed it locally (tests missing). I will fix this in the upcoming version
